contents/blog/running-group-targeted-ab-tests.md

```diff
@@ -0,0 +1,106 @@
+
+### 2. Higher randomization risk
+
+Randomization risk can occur when groups are not properly randomized when assigned to the control or test variants. Since you have fewer data points in group-targeted experiments, the risk is higher that this will occur, and it can distort results.
+
+For example, say a B2B SaaS company wants to see if a new pricing plan helps retain customers. They split companies into two groups: one with the new plan and one with the old plan. However, the randomization process assigns more small companies to the new plan and more large companies to the old plan. This may skew the results because company size could be a factor in determining customer retention rather than the pricing plan itself.
+
+### 3. Fewer user-level insights
+
+Since group-targeted experiments provide insights on an aggregate level, they may not show insights on a user level.
+
+For example, say a B2B SaaS tool wants users to invite coworkers to use the app. They add a big "Invite Coworker" button and test it in a company-targeted experiment. 
+
+The results show an increase in invites sent, but it's hard to know how individual users used the button. For example, what are the roles or skill sets of team members who are most likely to interact with the button and send invites?
+
+## Combining user-targeted and group-targeted experiments to reduce risk
+
+To improve the quality of insights from group-targeted experiments and address the risks mentioned above, you may want to consider running an experiment with your change at a user-level before running it at a group-level. This can help you in the following ways:
+
+### 1. Save time for changes that don't have an impact
+
+As mentioned, lower statistical power in group-targeted experiments means you need to run your experiments longer. However, a change that doesn't impact behavior at a user-level won't, in all likelihood, impact behavior at a group-level either.
+
+To rule this out, consider running a user-targeted experiment first in order to see if there is any impact on individual user behavior (where it should be quicker to gain results from because of higher statistical power) before running a group-targeted experiment.
+
+### 2. Gain user-level insights and identify randomization risks
+
+By running a user-targeted experiment first, you'll gain insight into how individual users interact with your change and what characteristics they may possess. This will then give you an understanding of what results you can expect from a group-targeted experiment.
+
+This can also help you identify randomization risks. For example, let's say a B2B SaaS app runs a user-targeted experiment and notices that users in large companies are more likely to interact with their new feature. In this case, when they run a group-targeted experiment, they can ensure to filter out small and medium-sized companies in their experiment in order to obtain more accurate results.
```

Comment:
```suggestion
Running a user-targeted experiment first gives you insight into how individual users interact with your change and what properties they possess. This will then give you an understanding of what results you can expect from a group-targeted experiment.

This can also help you identify randomization risks. For example, let's say a B2B SaaS app runs a user-targeted experiment, and notices that users in large companies are more likely to interact with their new feature. In this case, when they run a group-targeted experiment, they can ensure to filter out small and medium-sized companies in their experiment in order to obtain more accurate results.
```